include/swift/ABI/Metadata.h

```diff
@@ -3358,8 +3391,10 @@ struct TargetInPlaceValueMetadataInitialization {
 
   TargetMetadata<Runtime> *allocate(
       const TargetTypeContextDescriptor<Runtime> *description) const {
-    if (hasRelocationFunction(description))
-      return RelocationFunction(description);
+    if (hasRelocationFunction(description)) {
```

Comment:
`hasRelocationFunction` seems misnamed now; the relocation function is a small aspect of a much bigger picture.